source/MLKit/Core/Enums.cs

```diff
@@ -4,8 +4,15 @@
 namespace MLKit.Core {
 	[Native]
 	public enum ImageSourceType : long {
-		ImageSourceTypeImage = 0,
-		ImageSourceTypePixelBuffer = 1,
-		ImageSourceTypeSampleBuffer = 2
+		Image = 0,
```

Comment:
I don't think we should be renaming these. Aside from it being a breaking change, the original names are accurate to the ObjectiveC naming

Reply:
Yeah, this one is on me, I was experimenting with Copilot assisting with validation that things matched the headers and it suggested I make the switch, never went back and verified against the header itself.